Components/Text-Utilities/Text-Utilities.css

```diff
@@ -0,0 +1,72 @@
+ .component-example{
+    flex-direction: column;
+    color: aliceblue;
+  }
+  
+  .h1,
+  .h2,
+  .h3,
+  .h4,
+  .h5,
+  .h6 {
+    font-family: "Roboto Slab", serif;
+    font-weight: bold;
+    display: block;
+    color: aliceblue;
+  }
+  
+  .h1 {
+    font-size: 2rem;
+  }
+  .h2 {
+    font-size: 2.5rem;
+  }
+  .h3 {
+    font-size: 2.17rem;
+  }
+  .h4 {
+    font-size: 1rem;
+  }
+  .h5 {
+    font-size: 0.83rem;
+  }
+  .h6 {
+    font-size: 0.67rem;
```

Comment:
Instead of applying the properties directly to the element, you can assign a class to it and then use that class name to give properties